TramsDataApi/Controllers/V2/FSSProjectController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System.Linq;
+using System.Text.Json;
+using TramsDataApi.ResponseModels;
+using TramsDataApi.UseCases;
+
+namespace TramsDataApi.Controllers.V2
+{
+    [ApiVersion("2.0")]
+    [ApiController]
+    [Route("v{version:apiVersion}/fss/")]
+    public class FssProjectController : Controller
+    {
+        private readonly ILogger<FssProjectController> _logger;
+        private readonly IGetAllFssProject _getAllFssProject;
+
+        public FssProjectController(ILogger<FssProjectController> logger, IGetAllFssProject getAllFssProject)
```

Comment:
should this` IGetAllFssProject getAllFssProject` be `IGetAllFssProjects fssProjects`